### PR TITLE
Update rubocop to a slightly less ancient version to allow us to use ruby-2.7 syntax

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
@ofedoren not great, I admit, but I don't really have the determination to fix all gorillion cops that were added or changed since rubocop-0.39.0. On the brighter side, this should allow us to use the triple dot splat.

File: lib/dynflow.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
+
 require 'apipie-params'
 require 'algebrick'
-require 'thread'
```
Comment (ofedoren) [inline lib/dynflow.rb:4]: Just to make sure, this isn't being used, is it?

**Review by ofedoren — COMMENTED** [review]
Thanks, @adamruzicka, love the changes :) The only question though:

File: lib/dynflow.rb (hunk 1 above)
Comment (adamruzicka) [reply lib/dynflow.rb:4]: We're using threads, but it seems to be auto-required. Or to put it the other way around, we don't seem to be using anything related to threads that we'd have to explicitly require

**Review by ofedoren — APPROVED** [review]
Thanks, @adamruzicka, LGTM :)

**Pull request merged**